Formula/ssh-agent-filter.rb

```diff
@@ -0,0 +1,61 @@
+class SshAgentFilter < Formula
+  desc "filtering proxy for ssh-agent"
+  homepage "https://github.com/tiwe-de/ssh-agent-filter"
+  url "https://github.com/tiwe-de/ssh-agent-filter/archive/0.4.2.tar.gz"
+  sha256 "5586e40da34f4afd873d7ff30f115c4a1b9455c1b81f3c4679c995db7615b080"
+
+  depends_on "help2man" => :build
+  depends_on "boost"
+  depends_on "nettle"
+
+  patch do
+    # upstream didn't want this because realpath isn't included in coreutils for Debian wheezy.
+    # see https://github.com/tiwe-de/ssh-agent-filter/pull/5
```

Comment:
I've commented there but we need to figure out how to get something merged there or can't accept this (as it shows upstream doesn't want to support macOS).